.eslintrc

```diff
@@ -0,0 +1,33 @@
+---
+
+root: true
+
+env:
+  browser: true
+  mocha: true
+  node: true
+
+rules:
+  comma-dangle: 0
```

Comment:
Trailing commas already exist in the code base, all non-discontinued major browser (well, except IE8 which will be discontinued in January next year) supports it as well.

I really like trailing commas, as it makes cleaner diffs like these possible for adding/removing stuff;
```diff
diff --git a/foo.js b/foo.js
index 9c78d86..eb26d74 100644
--- a/foo.js
+++ b/foo.js
@@ -2,4 +2,5 @@ var foo = [
 	"foo",
 	"bar",
 	"quux",
+	"baz",
 ];
```
vs
```diff
diff --git a/foo.js b/foo.js
index 67d2cbd..2aada49 100644
--- a/foo.js
+++ b/foo.js
@@ -1,5 +1,6 @@
 var foo = [
 	"foo",
 	"bar",
-	"quux"
+	"quux",
+	"baz"
 ];
```